packages/server/postgres/migrations/2025-03-18T15:55:35.723Z_normalizeUserTms.ts

```diff
@@ -0,0 +1,26 @@
+import {sql, type Kysely} from 'kysely'
+
+// biome-ignore lint/suspicious/noExplicitAny: `any` is required here since migrations should be frozen in time. alternatively, keep a "snapshot" db interface.
+export async function up(db: Kysely<any>): Promise<void> {
+  await db.schema.alterTable('User').dropColumn('tms').execute()
+}
+
+// biome-ignore lint/suspicious/noExplicitAny: `any` is required here since migrations should be frozen in time. alternatively, keep a "snapshot" db interface.
+export async function down(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .alterTable('User')
+    .addColumn('tms', sql`character varying(100)[]`, (col) => col.defaultTo('{}').notNull())
+    .execute()
+  await db
+    .updateTable('User')
+    .set((eb) => ({
+      tms: eb.fn.coalesce(
```

Comment:
question-- i don't understand how this PR works 😅 we're setting the tms in a migration, but have removed all the places where we update the tms column?

Reply:
ah! dumb me. ok